entry_types/scrolled/app/models/pageflow_scrolled/chapter.rb

```diff
@@ -3,6 +3,30 @@ class Chapter < Pageflow::ApplicationRecord
     include Pageflow::SerializedConfiguration
     include Pageflow::AutoGeneratedPermaId
 
-    has_many :sections, -> { order('position ASC') }, dependent: :destroy, inverse_of: :chapter
+    belongs_to :storyline, touch: true
+    has_many :sections,
+             -> { order('pageflow_scrolled_sections.position ASC') },
+             dependent: :destroy,
+             inverse_of: :chapter
+    has_many :content_elements, through: :sections
+
+    delegate :entry, to: :storyline
```

Comment:
If I'm not missing something, these delegates are currently only used in tests. In the paged `Chapter` and `Storyline` models they were added as sort of last resort when first the `Revision` model was inserted between `Chapter` and `Entry` and later the `Storyline` model was placed between `Chapter` and `Revsion` to keep calls of the form `chapter.entry` working. In general I'd consider them an anti-pattern, though, since they lead to a series of DB queries which are often hard to eager load to prevent N+1 situations.

In the tests we can use transient factory attributes ([example](https://github.com/codevise/pageflow/blob/d5fd99fa09503bc3aa3d9326b242e7723109cfa1/spec/factories/pages.rb#L13), [example](https://github.com/codevise/pageflow/blob/d5fd99fa09503bc3aa3d9326b242e7723109cfa1/spec/factories/chapters.rb#L11)) to accomplish the same thing.